recipes/source/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "source" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fair-ease/Source/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b92fd32f80d6f894da9f130b792fddb3a6f0e1b56aca4d01baa653ed7bcbc1ff
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry-core
+
+  run:
+    - python >=3.8
+    - pykml
+    - lxml
+    - numpy
+    - pandas
+    - netcdf4
+    - seawater
```

Comment:
@Marie59 sorry for the off-topic comment but as the author of seawater it scares me a bit when folks are using it, b/c it is based on the deprecated EOS-80, instead of the of the new gsw which is based on the new TEOS-10. I'm sure there are legacy reasons and/or functions that only exists on seawater but if that is not the case, let me know if I can help you update to gsw.

Reply:
@ocefpaf oh ok, I am not the author of Source but I will convey your warning/recommendations about seawater to the them. Thank you for your help !

Reply:
They probably already know but send this repo to them: https://github.com/TEOS-10/GSW-Python

And let them know I'm available to answer any questions. Thanks!